app/qml/ProjectPanel.qml

```diff
@@ -434,11 +436,9 @@ Item {
           if (__merginApi.apiVersionStatus === MerginApiStatus.OK && stackView.currentItem.objectName === "authPanel") {
             if (__merginApi.userAuth.hasAuthData()) {
               refreshProjectList()
-            } else if (toolbar.highlighted !== homeBtn.text) {
-              if (stackView.currentItem.objectName !== "authPanel") {
+            } else if (stackView.currentItem.objectName !== "authPanel") {
                 stackView.push(authPanelComp, {state: "login"})
               }
-            }
```

Comment:
what is this change related to?

Reply:
This is just fixed leftover from refactoring - `toolbar` component doesn't exists. I will it move to a separate PR.